modules/extensions-spring-support/pom.xml

```diff
@@ -432,5 +432,10 @@
             <classifier>mule-plugin</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
```

Comment:
this is already included: https://github.com/mulesoft/mule/blob/master/modules/extensions-spring-support/pom.xml#L153

Reply:
it is necessary to add it with test scope, otherwise PetStoreCachedConnectionWithDynamicParameterTestCase fails